.travis.yml

```diff
@@ -50,5 +50,3 @@ jobs:
       env: PURPOSE='SyncAvailableExtensionsDoc'
       if: repo = Azure/azure-cli-extensions and branch = master and type = cron
   fast_finish: true
-  allow_failures:
```

Comment:
Is this intentional?

Reply:
https://github.com/Azure/azure-cli-extensions/issues/1067  by convenience. Leave the open will introduce more confusion.